torch/csrc/jit/python/script_init.cpp

```diff
@@ -1593,6 +1599,16 @@ void initJitScriptBindings(PyObject* module) {
       py::arg("force_outplace"),
       py::arg("argument_names") = std::vector<std::string>());
 
+  m.def("_compile_graph_to_code_table",
+        [](
+          const std::string& name,
+          const std::shared_ptr<Graph>& graph) {
```

Comment:
here it doesnt have compilation options as declared in the .pyi.in file

Reply:
Yep good catch!